test/testdata/generate.sh

```diff
@@ -57,7 +57,7 @@ function scaffold_test_project {
     if [ $project == "project-v2" ]; then
       $kb create api --plugins="go/v2,declarative" --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
```

Comment:
Is there a reason `declarative` is here too?
```suggestion
      $kb create api --plugins="go/v2" --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
```

Reply:
Sry, that was a mistake! I shouldn't have removed from `go/v3` either. Added it back and regenerated the samples.